table/tables/mutation_checker.go

```diff
@@ -295,6 +302,7 @@ func collectTableMutationsFromBufferStage(t *TableCommon, memBuffer kv.MemBuffer
 // Returns error if the index data is not a subset of the input data.
 func compareIndexData(
 	sc *stmtctx.StatementContext, cols []*table.Column, indexData, input []types.Datum, indexInfo *model.IndexInfo,
+	tableName string,
```

Comment:
We could pass table info directly as something related to the table could be referenced in the future for example if we want to print out if it is partitioned table etc.

Reply:
I think the name is the only info we need at present, and it's easy to change the signature in the future if needed.
Anyway I've made this change. There's no big difference